packages/ui/src/contexts/shortcuts.tsx

```diff
@@ -30,6 +30,8 @@ const InitialShortcuts: ShortcutsByModule = {
     {key: '=', action: 'Zoom in'},
     {key: '-', action: 'Zoom out'},
     {key: "'", action: 'Toggle grid'},
+    {key: 'v', action: 'Toggle coordinates'},
+    {key: 'c', action: 'Copy coordinates'},
```

Comment:
I'm not a huge fan of adding multiple shortcuts to do one thing.
I'd imagine it working the same way as the color picker, so:
1. **Initial State** - the coordinates are displayed in the bottom left, next to a button
    a. Clicking on the button switches to the **Sampling State**
    b. Pressing `v` switches to the **Sampling State**
3. **Sampling State**
    a. Clicking LMB anywhere on the canvas copies the position.
    b. Clicking ESC cancels sampling.

Reply:
I just thought I would add a toggle to turn the coordinates off in case someone did not want to see them. Do you want the coordinates always displayed?

Reply:
Yes, we could also go this way.
If you think hiding the coordinates can be useful then we can add it as a checkbox to the application settings.

Reply:
I added the coordinates as a toggle in the settings. I did not make it a select button like the color picker. Currently its just a keybind to copy the coordinates. This is kind of a stretch goal for me if I have the time.

Reply:
@aarthificial Couple clarity questions if you want it to be a sampling state instead.

For the sampling state do you want the same EyeDropper selection as the color picker? Or do you want something different?

What svg icon would you like for the button itself?